WMTSOptions default values

vcoindet opened the pull request:
WMTSOptions default values in WMTS_Provider.js

options.version = options.version || "1.0.0";
options.tileMatrixSet = options.tileMatrixSet || "WGS84";
options.mimetype = options.mimetype || "image/png";
options.style = options.style || "normal";
options.projection = options.projection || "EPSG:3857";

vcoindet committed:
WMTSOpotions in WMTS_Provider.js

options.version = options.version || "1.0.0";
options.tileMatrixSet = options.tileMatrixSet || "WGS84";
options.mimetype = options.mimetype || "image/png";
options.style = options.style || "normal";
options.projection = options.projection || "EPSG:3857";

peppsac commented on src/Core/Commander/Providers/WMS_Provider.js:
```diff
@@ -76,7 +76,7 @@ WMS_Provider.prototype.preprocessDataLayer = function(layer){
     layer.width = defaultValue(layer.heightMapWidth, 256),
     layer.version = defaultValue(layer.version, "1.3.0"),
-    layer.styleName = defaultValue(layer.style, "normal"),
+    layer.style = defaultValue(layer.style, "normal"),
```
Maybe you can drop the `defaultValue` usage so WMS and WMTS look similar

vcoindet replied:
ok, I'm on it

vcoindet replied:
done

vcoindet committed:
WMTSOptions default values in WMTS_Provider.js

options.version = options.version || "1.0.0";
options.tileMatrixSet = options.tileMatrixSet || "WGS84";
options.mimetype = options.mimetype || "image/png";
options.style = options.style || "normal";
options.projection = options.projection || "EPSG:3857";

WMSOptions default values in WMS_Providers

layer.format = layer.options.mimetype || "image/png";
layer.crs = layer.projection || "EPSG:4326";
layer.width = layer.heightMapWidth || 256;
layer.version = layer.version || "1.3.0";
layer.style = layer.style || "normal";
layer.transparent = layer.transparent || false;layer.bbox = layer.bbox || new BoundingBox();

vcoindet commented:
Branch updated and rebased
